### PR TITLE
add more words to spell check

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -451,7 +451,7 @@ IF (WIN32)
     SET (DEFAULT_BIN_SUBDIR bin)
     SET (DEFAULT_CGIBIN_SUBDIR bin)
     # put all the build products into a single directory
-    # under build (doesnt affect install target) to make for
+    # under build (doesn't affect install target) to make for
```
Comment (sbrunner) [inline CMakeLists.txt]: Not sure that's good, normally we shouldn't use the short form on writing...

File: ChangeLog [hunk 2]
```diff
@@ -155336,7 +155336,7 @@ gsherman <gsherman@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2006-11-27
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2006-11-27
 
-    Force tab bar scrolling on mac. For me it doesnt make the scroll icons appear but at least teh tabbar doesnt autoexpand any more
+    Force tab bar scrolling on mac. For me it doesn't make the scroll icons appear but at least teh tabbar doesn't autoexpand any more
```
Comment (DelazJ) [inline ChangeLog]: teh

File: ChangeLog [hunk 3]
```diff
@@ -167587,7 +167587,7 @@ gsherman <gsherman@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-10-06
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-10-05
 
     Apply wonders make patch for geos issues
-    Try to improve errror checking for when ~/.qgis.db doesnt exist
+    Try to improve errror checking for when ~/.qgis.db doesn't exist
```
Comment (DelazJ) [inline ChangeLog]: errror

File: ChangeLog [hunk 4]
```diff
@@ -170437,7 +170437,7 @@ mhugent <mhugent@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-04-26
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-04-25
 
-    Prevent assert failure when adding pg layer...still doesnt seem to set layer proejction properly though...weill investigate further tomorrow.
+    Prevent assert failure when adding pg layer...still doesn't seem to set layer proejction properly though...weill investigate further tomorrow.
```
Comment (DelazJ) [inline ChangeLog]: proejction
weill?

File: ChangeLog [hunk 5]
```diff
@@ -173952,7 +173952,7 @@ timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-01-01
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2005-01-01
 
-    Initial commit of projextion integration.....dont get too happy cause its still not working yet...though it behaves nicely and doesnt kill qgis if it cant set up the projection environment properly. I believe the problem may be resolved by linking to libproj during build, because currently I get errors like this:
+    Initial commit of projextion integration.....dont get too happy cause its still not working yet...though it behaves nicely and doesn't kill qgis if it cant set up the projection environment properly. I believe the problem may be resolved by linking to libproj during build, because currently I get errors like this:
```
Comment (DelazJ) [inline ChangeLog]: Some more...
* **dont** get
* **cant**

File: ChangeLog [hunk 6]
```diff
@@ -174961,7 +174961,7 @@ gsherman <gsherman@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-11-20
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-11-20
 
-    Accidentally committed this earlier but its has dodgy multipart poly labelling stuff that wasnt meant to go into cvs yet so its commented out for now....
+    Accidentally committed this earlier but its has dodgy multipart poly labelling stuff that wasn't meant to go into cvs yet so its commented out for now....
```
Comment (DelazJ) [inline ChangeLog]: labelling (weird, this one was already referenced)

File: ChangeLog [hunk 7]
```diff
@@ -176403,7 +176403,7 @@ timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-09-21
 
     Fix for clipping of labels when labelling polygon features.
 
-    Modified label rendering procedure so that it is a seprate call (drawLabels) and no longer part of the draw() method. Added a second loop in QgsMapCanvas::render() to go through all visible layers and call their drawLabels method. This loop is called after the features rendereing loop so the clipping problem is thusly forthwithly avaoided. Performance doesnt seem to bad either.
+    Modified label rendering procedure so that it is a seprate call (drawLabels) and no longer part of the draw() method. Added a second loop in QgsMapCanvas::render() to go through all visible layers and call their drawLabels method. This loop is called after the features rendereing loop so the clipping problem is thusly forthwithly avaoided. Performance doesn't seem to bad either.
```
Comment (DelazJ) [inline ChangeLog]: seprate
rendereing
avaoided

File: ChangeLog [hunk 8]
```diff
@@ -175624,7 +175624,7 @@ timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-11-03
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-11-03
 
-    First part of scale dependent layer visibility. Currently its not linked up to the layer properties gui (thats the second part) and it doesnt save/restore state when project is saved/loaded (thats the third part).
+    First part of scale dependent layer visibility. Currently its not linked up to the layer properties gui (thats the second part) and it doesn't save/restore state when project is saved/loaded (thats the third part).
```
Comment (DelazJ) [inline ChangeLog]: thats

File: ChangeLog [hunk 9]
```diff
@@ -178230,7 +178230,7 @@ timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-06-20
 
     ** Fix for bug [ 973922 ] Overview shows layers in wrong order
 
-    ** Fixed show stopper bug where maplayerregistry wasnt being cleared properly on file new
+    ** Fixed show stopper bug where maplayerregistry wasn't being cleared properly on file new
 
     ** Added setZOrder which will be used in next commit to fix projectio zorder problem
```
Comment (DelazJ) [inline ChangeLog]: projectio

File: ChangeLog [hunk 10]
```diff
@@ -178420,7 +178420,7 @@ timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-06-16
 
 timlinux <timlinux@c8812cc2-4d05-0410-92ff-de0c093fc19c>	2004-06-16
 
-    Beginnings of generic vector file writer - incomplete and doesnt do anything useful yet except has abilty to make a shapefile with a couple of user defined fields e.g. to create a new point shapefile:
+    Beginnings of generic vector file writer - incomplete and doesn't do anything useful yet except has abilty to make a shapefile with a couple of user defined fields e.g. to create a new point shapefile:
```
Comment (DelazJ) [inline ChangeLog]: abilty

File: CMakeLists.txt (hunk 1 above)
Comment (3nids) [inline CMakeLists.txt]: Be careful,  you just did ;) 
More seriously, some of  these were already in the dictionary,  I believe they are accepted nowadays. Moreover, they are still better than something which doesn't exist.

**Comment by 3nids** [comment]
Thanks a lot @DelazJ will add them

**Comment by DelazJ** [comment]
@3nids I added more words for spelling check.
I didn't know that the changelog file could/should also be fixed afterward. I asked at https://github.com/qgis/qgis3_UIX_discussion/issues/19#issuecomment-268365999 and didn't get an answer.

File: CMakeLists.txt (hunk 1 above)
Comment (sbrunner) [inline CMakeLists.txt]: > than something which doesn't exist.
Effectively yes, for the rest I don't know how it's accepted or not then don't worry...

**Comment by 3nids** [comment]
@DelazJ thanks for the check. I removed the Changelog from the corrected file. Not sure what's best either.

**Pull request merged**